.github/workflows/tests.yml

```diff
@@ -140,26 +140,29 @@ jobs:
         with:
           file: "pyproject.toml"
           field: "tool.typeshed.pyright_version"
-      - uses: jakebailey/pyright-action@v1
+      - name: Run pyright with basic settings on all the stubs
+        uses: jakebailey/pyright-action@v1
         with:
           version: ${{ steps.pyright_version.outputs.value }}
           python-platform: ${{ matrix.python-platform }}
           python-version: ${{ matrix.python-version }}
           no-comments: ${{ matrix.python-version != '3.10' || matrix.python-platform != 'Linux' }} # Having each job create the same comment is too noisy.
-          project: ./pyrightconfig.stricter.json
```

Comment:
I think running stricter first would fail faster in most cases. Since they both run on all files not excluded by the stricter settings. Stricter runs on less files and a failure with basic config would also result in a failure with stricter ones.

But this job is also pretty fast anyway.

Reply:
Also I've found it pretty annoying in the past when we're working to enable new pyright lints, the way you have to play "pyright whackamole" with our current setup. You think you've got all the pyright errors sorted, but then you find out, oh no, it was only the strict-config errors you've got sorted! Most of typeshed is still to go, it's just we never even got to the second step 😶

This order feels more intuitive to me :)